### PR TITLE
Update src docs for Axis Gui Additional features

### PR DESCRIPTION
**Pull request by ArcEye**
Docs for ini_config and axis updated to reflect
additional settings available following recent
commit to main repo with same

Signed-off-by: Mick Grant <arceye@mgware.co.uk>

### CODE REVIEW

File: src/gui/axis.asciidoc [hunk 1]
```diff
@@ -964,4 +984,14 @@ preview on certain parts that are already working OK).
 This display can be useful in the Axis preview when (debug,message)
 comments are not displayed.
 
+*NB*
```
Comment (zultron) [inline src/gui/axis.asciidoc]: Could the `[NOTE]` syntax could be used here?  See the example above.

File: src/gui/axis.asciidoc [hunk 2]
```diff
@@ -964,4 +984,14 @@ preview on certain parts that are already working OK).
...
+The preview display can be completely disabled for all loaded files by
+an ini file field
+
+'[USER_COMMANDS]'
+
+DISABLE_DISPLAY = YES
+
```
Comment (zultron) [inline src/gui/axis.asciidoc]: Is this literal code from a `.ini` file?  Does it make sense to use a code block with `----`?  See the example above.

**Comment by zultron** [comment]
Looks fine to me, except the last hunk of `src/gui/axis.asciidoc` looks like an afterthought; see my per-line nit-picky comments above.

**Comment by luminize** [comment]
I think you comments and my merge crossed each other.
I'll make an issue of making `.asciidoc` writing preferences

**Comment by ArcEye** [comment]
"Could the [NOTE] syntax could be used here? See the example above."

I am sure it could, albeit NB seems to do the same thing.
I had a lot of trouble with this doc, followed the same style as previous entries and had bizarre errors where previous entries were no longer visible once I created a new one etc etc
Rather ran out of time and patience with it.

Still have at least 3 more docs to do, will see how they go!

**Comment by luminize** [comment]
> On 16 Jan 2015, at 17:50, ArcEye <notifications@github.com> wrote:
> 
> "Could the [NOTE] syntax could be used here? See the example above."
> 
> I am sure it could, albeit NB seems to do the same thing.
> I had a lot of trouble with this doc, followed the same style as previous entries and had bizarre errors where previous entries were no longer visible once I created a new one etc etc
> Rather ran out of time and patience with it.
> 
Do you mean changed files? I did a lot moving etc with them. Also the subtree stuff I struggled with when splitting. Might your missing entries be caused by me?

**Comment by ArcEye** [comment]
Nothing you did, this was whilst trying to edit the document in github
I used the same format of
.menuname
as a header that all the previous descriptions of menu entries used and not only did my header not show up, but all the previous ones disappeared too
I just gave up eventually and used
*menuname*
for all of them which seemed to work

**Pull request merged**